lib/sign_notarize.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -ex
+
+if [[ -z $WORKSPACE ]]; then
+  WORKSPACE=$PWD
+fi
+
+UNAME=`uname`
```

Comment:
uname not needed since we run on darwin only, which is invariant

Reply:
done